Gopkg.toml

```diff
@@ -8,23 +8,23 @@
 
 [[override]]
   name = "k8s.io/api"
-  version = "kubernetes-1.11.2"
+  version = "kubernetes-1.11.5"
 
 [[override]]
   name = "k8s.io/apimachinery"
-  version = "kubernetes-1.11.2"
+  version = "kubernetes-1.11.5"
 
 [[override]]
   name = "k8s.io/apiextensions-apiserver"
-  version = "kubernetes-1.11.2"
+  version = "kubernetes-1.11.5"
```

Comment:
There is no mention on why we bumped this to 1.11.5?

Reply:
No longer bumping this thanks to #807